traffic_portal/app/src/common/modules/form/deliveryService/form.deliveryService.DNS.tpl.html

```diff
@@ -78,7 +78,10 @@
                         <option ng-value="false">Not Active</option>
                     </select>
                     <small class="input-error" ng-show="hasPropertyError(deliveryServiceForm.active, 'required')">Required</small>
-                    <small class="input-diff" ng-if="settings.isRequest" ng-show="open() && deliveryService.active != dsCurrent.active">Current Value: [ {{dsCurrent.active ? 'Active' : 'Not Active'}} ]</small>
+                    <aside class="current-value" ng-if="settings.isRequest" ng-show="open() && deliveryService.active != dsCurrent.active">
+                        <label for="current-value-active">Current Value</label>
+                        <pre id="current-value-active">{{::dsCurrent.active ? 'Active' : 'Not Active'}}</pre>
```

Comment:
yeah, i can switch to <h{{n}}> but i guess i'll just pick `n` because there are no other instance of header tags used in the document except h6

Reply:
h3 i guess

Reply:
odd, I swear there was at least one. In general you're not supposed to skip levels, but I think it's fine since h1 and h2 don't really seem appropriate at that level.